lib/peep.rb

```diff
@@ -0,0 +1,11 @@
+require 'pg'
+
```

Comment:
The next step for this app would be to use your Peep class in app.rb... then think about adding the database